cmd/mapt/cmd/aws/hosts/mac.go

```diff
@@ -26,7 +26,7 @@ const (
 	archDefault       string = "m2"
 	osVersion         string = "version"
 	osVersionDesc     string = "macos operating system vestion 11, 12 on x86 and m1/m2; 13, 14 on all archs"
-	osDefault         string = "14"
+	osDefault         string = "15"
```

Comment:
seem we are duplicating the constants for default values of `arch` and `os` in `pkg/provider/aws/action/mac/constants.go`

Reply:
Ahh yeah..may we need to use those from action on the cmd ...as on actions we need them for sure